### PR TITLE
Plotly concepts express terms histogram

### LINKED ISSUE #4562: [Term Entry] Python Plotly .histogram()
### Reason/inspiration (optional)

We would like a new term entry in the `Data Visualization` concept for Plotly: `.histogram()`. The entry should go in a new file under `docs/content/plotly/concepts/express/terms/histogram/histogram.md`

The entry should describe the term and how it is used. Also include:

- An introductory paragraph that describes the concept
- A 'Syntax' section that includes the syntax for the concept (if applicable)
- An 'Example' section that shows the concept in use

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by romanwilk**
### Description

Created new documentation, introducing information about histogram term in Plotly library for Python.

### Issue Solved

Closes #4562 

### Type of Change
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/plotly/concepts/express/terms/histogram/histogram.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - 'Data'
+  - 'Graphs'
+  - 'Methods'
+  - 'Libraries'
+  - 'Plotly'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
```
Comment (avdhoottt) [inline content/plotly/concepts/express/terms/histogram/histogram.md:17]: Keep only 2 slugs:
- A free course
- A Pro course/Path

File: content/plotly/concepts/express/terms/histogram/histogram.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+In Plotly, the **`.histogram()`** method as a part of the `plotly.express` interface, is used to make a graphical representation of data as an accumulated bar chart where, each stocked data is counted and represented. This method offers several functions like sum, average, count and more, which can be used to visualize data on categorical and date axes or linear axes. The elements of histogram graph are:
```
Comment (avdhoottt) [inline content/plotly/concepts/express/terms/histogram/histogram.md:20]:
```suggestion
In Plotly, the `**.histogram()**` method in the `plotly.express` module is used to create a graphical representation of data as a bar chart that shows the distribution of a dataset. The bars represent the count of data points within specified ranges (bins). This method also allows for aggregation functions like sum, average, and count, which can be applied to visualize data on categorical, date, or linear axes. 

The elements of the histogram graph are:
```

File: content/plotly/concepts/express/terms/histogram/histogram.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ - The title: describes the information represented in the histogram.
+ - X-axis: intervals that show the scale of values which measures all fall under.
+ - Y-axis: the number of times that the values occurred within the intervals set by the X-axis.
+ - The bars: The height of the bar shows the number of times that the values occurred within the interval, width of the bar shows the interval that is covered.
+
+## Syntax
+
+```pseudo
+plotly.express.histogram(data_frame=None, x=None, y=None, color=None, pattern_shape=None, facet_row=None, facet_col=None, facet_col_wrap=0, facet_row_spacing=None, facet_col_spacing=None, hover_name=None, hover_data=None, animation_frame=None, animation_group=None, category_orders=None, labels=None, color_discrete_sequence=None, color_discrete_map=None, pattern_shape_sequence=None, pattern_shape_map=None, marginal=None, opacity=None, orientation=None, barmode='relative', barnorm=None, histnorm=None, log_x=False, log_y=False, range_x=None, range_y=None, histfunc=None, cumulative=None, nbins=None, text_auto=False, title=None, template=None, width=None, height=None)
```
Comment (avdhoottt) [inline content/plotly/concepts/express/terms/histogram/histogram.md:29]:
```suggestion
plotly.express.histogram(
    data_frame,    # Your DataFrame containing the data
    x,             # Column name for the x-axis (required)
    y=None,        # Optional column name for the y-axis
    color=None,    # Optional column name for color grouping
    facet_row=None,# Optional column name to split data into rows
    facet_col=None,# Optional column name to split data into columns
    hover_data=None,# Optional list of column names for hover info
    marginal=None, # Optional to add marginal plots (e.g., 'rug', 'box')
    opacity=None,  # Optional opacity for bars
    orientation=None,# 'v' or 'h' for vertical or horizontal bars
    barmode='relative', # Bar stacking mode: 'relative', 'overlay', etc.
    title=None,    # Optional title for the plot
    template=None, # Optional Plotly template (e.g., 'plotly_dark')
    width=None,    # Optional plot width
    height=None    # Optional plot height
)
```

File: content/plotly/concepts/express/terms/histogram/histogram.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import plotly.express as px
+
+# Loading a built-in data frame
+df = px.data.tips()
+
+# Creating the histogram with the total_boll as the x variable and integer "20" as nbins
+fig = px.histogram(df, x="total_bill", nbins=20)
+
+# Showing the histogram plot
+fig.show()
```
Comment (avdhoottt) [inline content/plotly/concepts/express/terms/histogram/histogram.md:53]:
```suggestion
# Importing plotly.express as px
import plotly.express as px

# Loading a built-in data frame
df = px.data.tips()

# Creating the histogram with "total_bill" as the x variable and integer 20 as nbins
fig = px.histogram(df, x="total_bill", nbins=20)

# Showing the histogram plot
fig.show()
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @romanwilk, I've suggested some changes, please make them. Thanks!!

File: content/plotly/concepts/express/terms/histogram/histogram.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  - 'Data'
+  - 'Graphs'
+  - 'Methods'
+  - 'Libraries'
+  - 'Plotly'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
```
Comment (avdhoottt) [inline content/plotly/concepts/express/terms/histogram/histogram.md:17]:
```suggestion
  - 'learn-python-3'
  - 'paths/computer-science'
```

**Review by Sriparno08 — APPROVED** [review]
Looks good, @romanwilk!

**Pull request merged**